src/main/java/hudson/plugins/ec2/win/EC2WindowsLauncher.java

```diff
@@ -73,8 +73,9 @@ protected void launchScript(EC2Computer computer, TaskListener listener) throws
             logger.println("remoting.jar sent remotely. Bootstrapping it");
 
             final String jvmopts = node.jvmopts;
+            final String remoteFS = node.getRemoteFS();
             final WindowsProcess process = connection.execute("java " + (jvmopts != null ? jvmopts : "") + " -jar "
-                    + tmpDir + AGENT_JAR, 86400);
+                    + tmpDir + AGENT_JAR + " -workDir " + remoteFS, 86400);
```

Comment:
Doesn't this parameter need to be escaped/quoted?

Reply:
Doesn't seem to need to be escaped in my testing it simply accepted it. Although it should require escaping in the case of paths with spaces